1-exercises/D-object-methods/exercise.js

```diff
@@ -9,6 +9,9 @@
 
 let student = {
   // write code here
+  getName: function(name){
```

Comment:
It is possible to use an arrow function here: let student = {getName: (name) => console.log(`Student name: ${name}`)}